benchmarks/perf-tool/sample-configs/faiss-nsg/index-spec.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
we should add similar files in release-configs folder too.

Reply:
++ @ https://github.com/opensearch-project/k-NN/pull/945/commits/819fcedce70fc452fb55152ab873a0e5dea53923